kod/util/factgame/territry.kod

```diff
@@ -396,18 +396,24 @@ messages:
       return;
    }
 
-   ReloadSavedFlagStates()
+   ReloadFlagStates(lFlagStates=$)
    {
       local i, iRID, oRoom, oFlag;
 
-      if plFlagStates = $
+      if lFlagStates = $
       {
-         DEBUG("ReloadSavedFlagStates:: nothing saved, nothing reloaded");
+         lFlagStates = plFlagStates;
```

Comment:
Having the caller always pass in lFlagStates would be better than this strange fallback.